app/action/responsibility/post.controller.js

```diff
@@ -0,0 +1,35 @@
+const { body } = require('express-validator')
+
+const errorMessage = 'Select the person responsible'
+const options = ['SERVICE_USER', 'PRACTITIONER', 'OTHER']
+
+const validationRules = () => {
+  return [
+    body('responsibility', errorMessage)
+      .not()
+      .isEmpty()
+      .bail()
+      .if(responsibility => typeof responsibility === 'string')
```

Comment:
Probably my dimness - what does this line give us that the following .isString() doesn't?

Reply:
The 'if' statement provides no validation itself as such although progression through it implies a validation of sorts. On reflection theoretically other types (object, int) could cause validation to fail though, so I'm going to rejig it slightly to catch these unlikely edge cases. So I'll change this to represent an if not an Array scenario.